pkg/storage/segments/store.go

```diff
@@ -192,7 +193,13 @@ func (s *segmentStore) Get(ctx context.Context, path paths.Path) (
 			return nil, Meta{}, Error.Wrap(err)
 		}
 
-		rr, err = s.ec.Get(ctx, nodes, s.rs, pid, pr.GetSize())
+		es, err := makeErasureScheme(pr.GetRemote().GetRedundancy().GetMinReq(),
+			pr.GetRemote().GetRedundancy().GetTotal(), pr.GetRemote().GetRedundancy().GetErasureShareSize())
```

Comment:
`makeEreasureScheme` would be even more helpful if it just take a single param `pr.GetRemote().GetRedundancy()`? It can get the min req, total and erasure share size from it in the function implementation.